web_ui/src/pages/annotator/annotation/annotation-list/annotation-list.module.scss

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2022-2025 Intel Corporation
+// LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
+
+div:has(> .dropTarget) {
```

Comment:
Can we add a class name here? I'm worried that this adds a global style that could affect other components.

Reply:
`dropTarget` is the custom class name: https://github.com/open-edge-platform/geti/pull/144/files#diff-8b288f61dd0edda47f2d9de8a252730353cc42cf67b0520e4705fb52cdea072fR44